sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md

```diff
@@ -110,20 +110,19 @@ Refer to [Logging](#logging) to enable loggers for related libraries.
 ## Next steps
 
 ### Examples
-- [./examples/eventprocessor/event_processor_blob_storage_example.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/examples/event_processor_blob_storage_example.py) - event processor with blob partition manager example
+- [./samples/event_processor_blob_storage_example.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/samples/event_processor_blob_storage_example.py) - EventHubConsumerClient with blob partition manager example
 
 ### Documentation
 
-Reference documentation is available at https://azure.github.io/azure-sdk-for-python/ref/azure.eventhub.extensions.html.
+Reference documentation is available at https://azuresdkdocs.blob.core.windows.net/$web/python/azure-eventhub/5.0.0b5/azure.eventhub.extensions.html
```

Comment:
Extensions being empty, this link will not exits. What do you wanted to show here?

Reply:
I have updated the link , pointing to the `PartitionManager` in eventhub docs

Reply:
I think it's due to the github that can't show the changes.

I checked files in the branch and see the links are up to date.

[README of azure-eventhubs-checkpointstoreblob](
https://github.com/Azure/azure-sdk-for-python/blob/feature/eventhubs_preview5/sdk/eventhub/azure-eventhubs-checkpointstoreblob/README.md)

[README of azure-eventhubs-checkpointstoreblob-aio](
https://github.com/Azure/azure-sdk-for-python/blob/feature/eventhubs_preview5/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md)